pkg/signup/signup_service_test.go

```diff
@@ -39,7 +39,7 @@ func (s *TestSignupServiceSuite) TestCreateUserSignup() {
 	userID, err := uuid.NewV4()
 	require.NoError(s.T(), err)
 
-	userSignup, err := svc.CreateUserSignup("jsmith", userID.String())
+	userSignup, err := svc.CreateUserSignup("jsmith", userID.String(), "jsmith@gmail.com")
```

Comment:
Also in the end of this test (line 61) check that the annotation is present in the signup you get from the fake client.

Reply:
done: c9c8d71e8c9a03b9177866327e11a54f3b86c95f